src/jit/registerfp.cpp

```diff
@@ -229,7 +229,12 @@ void CodeGen::genFloatSimple(GenTree *tree, RegSet::RegisterPreference *pref)
             genCodeForTreeFloat(op1, pref);
 
             inst_RV_TT(ins_FloatConv(tree->TypeGet(), op1->TypeGet()), REG_FLOATRET, op1);
-            if (compiler->info.compIsVarArgs)
+
+            bool UseIntReg = compiler->info.compIsVarArgs;
```

Comment:
Same as above to avoid compiler warning, I'd better write it as:
#ifndef ARM_SOFTFP
  if (compiler->info.compIsVarArgs)
#endif
  {
    ...